### ISSUE TITLE
Wav 2 GTA V Legacy PS2

### ISSUE THREAD

**Issue by insouso** [IsseiYoshida/Instalador-POPStarter#3]
gostaria de saber se tem como você de alguma forma me liberar esse programa que se usa em conjunto com o MFaudio, pois estou trabalhando em mods e ele ajudaria muito mesmo, mas apenas se estiver tudo bem liberar ele.

**Comment by IsseiYoshida** [comment]
Claro, só preciso procurar o código-fonte. Acho que ele está no meu sótão, em um dos HDs antigos.

**Comment by IsseiYoshida** [comment]
https://github.com/IsseiYoshida/Wav-2-Vag/releases/tag/Wav2Vag
